app/renderer/components/window.js

```diff
@@ -45,6 +47,7 @@ class Window extends React.Component {
 
     this.onChange = this.onChange.bind(this)
     this.onAppStateChange = this.onAppStateChange.bind(this)
+    this.updateComputedStyles = throttle(this.updateComputedStyles.bind(this), 66)
```

Comment:
I think we can get rid of this and rely on the existing `windowUpdated` debounce